Feature set up GitHub actions

mansa-susa opened the pull request:
- Set up Github Actions with the yaml file
-  Set up the html and css linter configurations
-  Add index html page with an H1 tag
-  Add style to the H1 tag on the stylesheet
-  Add .gitignore file and add the test.md and node_modules to it

mansa-susa committed:
Set up the github actions with the yaml configuration and the html and css linters config.

mansa-susa committed:
add the .gitignore, test.md and stylesheet files

mansa-susa committed:
Add the H1 tag to the index page and css declaration to the stylesheet

mansa-susa committed:
Add test.md and node_modules to the .gitignore file

mansa-susa committed:
Add the link tag to the index page to link the stylesheet

mansa-susa committed:
Edit the href attribute value to the correct stylesheet filename

mansa-susa committed:
fix linter error

mansa-susa committed:
fix linter error

mansa-susa committed:
Document project with ReadMe file

TadesseDev commented on README.md:
```diff
@@ -1 +1,189 @@
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+      This project will not have any additional features.
+
```
- [ ] You have done an incredible job @mansa-susa, to ensure everything is clean. Please, remove the `> Describe 1 - 3 features you will add to the project.` text from your readme. This is only a direction set by Microverse on how to use the project and should not be part of your readme file. 
- [ ] Please add two more `Future Features` to your project. The section `Future Features`, is expected to include 1-3 potential features to be added.

TadesseDev reviewed (changes requested):
### Hi @mansa-susa 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7](https://user-images.githubusercontent.com/69077061/212385281-4797c58a-d358-423a-aa7c-832c22203865.gif)




## Highlights
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mansa-susa committed:
Update README.md

- remove the > Describe 1 - 3 features you will add to the project.
- new feature that can be added is adding a title to the page

taldr27 reviewed (approved):
## Hi @mansa-susa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

![congrats](https://media.giphy.com/media/3o6fIUZTTDl0IDjbZS/giphy.gif)

## Highlights 🎉
✅ Professional Readme. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.

You can also consider:
- [ ] **[OPTIONAL]** Adding more **future features** to your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_
